src/store/config/state.ts

```diff
@@ -116,6 +116,14 @@ export const defaultState = (): ConfigState => {
         job_queue: [
           { value: 'time_added', visible: true },
           { value: 'time_in_queue', visible: false }
+        ],
+        spoolman: [
+          { value: 'filament_name', visible: true },
+          { value: 'id', visible: true },
+          { value: 'material', visible: true },
+          { value: 'location', visible: true },
+          { value: 'comment', visible: true },
+          { value: 'last_used', visible: true }
```

Comment:
Not sure these are necessary, I think they're just for hidden-by-default columns?

Reply:
Indeed, all we need here is the hidden columns, so I would revert the changes on this file.

Reply:
I stand corrected: the entry IS required, but only needs the columns with `visible:false`, the rest is not required.

Given this, using `[]` should be ok!